temporalnexus/operation.go

```diff
@@ -52,6 +56,18 @@ import (
 	"go.temporal.io/sdk/workflow"
 )
 
+// tokenSource is used as the "source" operation token header.
+const tokenSource = "io.temporal.sdk"
+
+// workflowRunOperationTokenType is used as the "type" operation token header for workflow run operations.
+const workflowRunOperationTokenType = "WorkflowRunOperation"
+
+// workflowRunOperationToken is the decoded form of the workflow run operation token.
+type workflowRunOperationToken struct {
+	NamespaceName string `json:"ns"`
```

Comment:
Is this format expected to be consistent across SDKs?

Reply:
I'd expect it to. Happy for others to chime in.